cli/src/config.rs

```diff
@@ -46,6 +46,7 @@ pub struct ProjectConfig {
     pub id: ProjectId,
     pub name: String,
     pub created_at: DateTime<Local>,
+    pub group_name: Option<String>,
```

Comment:
I don't think this is relevant? This is a file we're reading from the filesystem. When reading it the `Option` automatically parses it as `None` when it is missing. The only difference here would be that we explicitly write it to the config file when it is `None`, but this isn't a problem because if we do, our CLI obviously supports the groups feature already?

Reply:
The relevant situation is when the `.phylum_project` file is created by a newer version of the CLI and then committed to VCS and later read by an old version of the CLI.

It may be a rare situation, but there's still no reason to break it.

Reply:
This will not break it, as you can test by yourself by adding a random unknown key to the `.phylum_project` file. And a file created by a newer version of the CLI could otherwise still easily break old CLI versions when the group is present.